packages/api/src/command/medical/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -13,35 +19,38 @@ export type XCUrls = {
 
 export function parseCQDirectoryEntries(orgsInput: Organization[]): CQDirectoryEntryData[] {
   const orgs = orgsInput.flatMap(org => {
-    const orgOid = org?.identifier?.value?.value;
+    if (!org) {
+      return [];
+    }
+    const orgOid = org.identifier?.value?.value;
     if (!orgOid) return [];
 
-    const url = getUrls(org?.contained);
+    const url = getUrls(org.contained);
     if (!url?.urlXCPD) return [];
-    let oid = org?.identifier?.value?.value;
+    const oid = org.identifier?.value?.value;
     try {
-      oid = normalizeOid(org?.identifier?.value?.value);
+      const normalizedOid = normalizeOid(oid);
+      const coordinates = getCoordinates(org.address);
+      const state = getState(org.address);
+
+      const orgData: CQDirectoryEntryData = {
+        oid: normalizedOid,
+        name: org.name?.value ?? undefined,
+        urlXCPD: url.urlXCPD,
+        urlDQ: url.urlDQ,
+        urlDR: url.urlDR,
+        lat: coordinates?.lat ?? undefined,
+        lon: coordinates?.lon ?? undefined,
+        data: {
+          ...org,
+        },
+        state,
+      };
+      return orgData;
     } catch (err) {
       console.log(`Organization ${org.name?.value} has invalid OID: ${oid}`);
```

Comment:
We moved a bunch of code inside the try/catch, but we should then update the error handling part - this is assuming the error is only related to OID. For example, if we make change `getCoordinates` to throw an error in the future, we likely won't be coming back to this, so the catch would become misleading.

IMO, it would be better to say we can't create a directory entry for this org's data, or have a try/catch specific to `normalizeOid()` like we had before.

If you wanted to get rid of variables in favor of constants - 🙏, could use the function approach:

```ts
const getOID = () => {
   try {
      const normalizedOid = normalizeOid(oid);
   } catch (err) {
      console.log(`Organization ${org.name?.value} has invalid OID: ${oid}`);
   }
}
const normalizedOid = getOID();
```